sites/pycharm-guide/contents/tutorials/intro-aws/index.md

```diff
@@ -0,0 +1,26 @@
+---
+
+### Introduction
+- Brief intro about AWS SAM
```

Comment:
We will need more material on this top-level page. I can write some if you'd like.

Reply:
Sure.

Reply:
Can you help me on this.